### DISCUSSION TITLE
Understanding of POD scheduling Requests & Limits

### DISCUSSION THREAD

**Discussion by mallikkml** [kubernetes/website#43561] [Advice on using this repository]
Hi, Reading documentation I got understanding of POD requests and limits but not very clear on below queries.

Experts, please help me here.

1. Does scheduler look for available compute on cluster for requests, limits, or both? my understanding is POD gets scheduled if requests are met on cluster.
2. On a 3-node cluster cumulative CPU availability is 1200m (from node1: 500, node2: 400 & node3: 300) in this situation what will happen if a new POD requests/needs 800m CPU - will it be scheduled?
3. What will happen for similar situation of #2 with memory.

Thanks in advance for helping here.

**Comment by divya-mohan0209** [comment]
Hello @mallikkml , this is not a support forum for user queries and is a discussion forum for community members to opine about issues related to Kubernetes documentation. Please consider raising your queries on a separate forum such as Stackoverflow or serverfault.
